### PR TITLE
Form integration

### LINKED ISSUE #2135: Integrate form editing into Camunda Modeler
__What should we do?__

* [x] A new tab exists that holds the form editor (https://github.com/camunda/camunda-modeler/issues/2149)
* [x] `.form` files are recognized by the editor and opened on OS double click (https://github.com/camunda/camunda-modeler/issues/2149)
* [x] Forms can be loaded, edited and saved
* [x] `form-js` editor is integrated (https://github.com/camunda/camunda-modeler/issues/2150)
----

Depends on https://github.com/bpmn-io/form-js/issues/23

### PR DESCRIPTION
**Pull request by nikku**
__Compatibility Checklist__

* [x] Works on Linux
* [x] Works on Mac OS
* [x] Works on Windows

---

Try it out via 

```
npx @bpmn-io/sr camunda/camunda-modeler#form-integration
```

---

Some screenshots:

image

image

image

---

Closes https://github.com/camunda/camunda-modeler/issues/2149

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
Double-clicking works for me on Mac ✔️ ✅ 

Kapture 2021-03-11 at 15 58 33

File: client/src/app/tabs/json/CodeMirror.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    this.refresh();
+  };
+
+  instance.detach = function() {
+    if (el.parentNode) {
+      el.parentNode.removeChild(el);
+    }
+  };
+
+  instance.importXML = function(json) {
```
Comment (pinussilvestrus) [inline client/src/app/tabs/json/CodeMirror.js:70]: that's beautiful 🦄

**Comment by barmac** [comment]
image

Nice!

**Comment by andreasgeier** [comment]
Find the correct naming "Form (Camunda Forms)" here: https://github.com/camunda/camunda-modeler/issues/2136

**Comment by nikku** [comment]
Thanks @andreasgeier. Will update the PR accordingly.

**Comment by nikku** [comment]
And the name of the tab?

**Comment by nikku** [comment]
Corrected naming via https://github.com/camunda/camunda-modeler/pull/2147/commits/7b080ffcccad1464d126cf73e272aaf2160fa54e. Update screenshots, too :arrow_up:.

**Review by andreasgeier — APPROVED** [review]
What I can see on the screenshots looks as expected. Greenlight from my perspective.

**Pull request merged**